app/screens/settings-screen/settings/account-pos.tsx

```diff
@@ -10,21 +10,19 @@ import { SettingsRow } from "../row"
 export const AccountPOS: React.FC = () => {
   const { appConfig } = useAppConfig()
   const posUrl = appConfig.galoyInstance.posUrl
-  const shortUrl = appConfig.galoyInstance.lnAddressHostname
 
   const { LL } = useI18nContext()
 
   const { data, loading } = useSettingsScreenQuery()
   if (!data?.me?.username) return <></>
 
   const pos = `${posUrl}/${data.me.username}`
-  const short = `${shortUrl}/${data.me.username}`
 
   return (
     <SettingsRow
       loading={loading}
       title={LL.SettingsScreen.pos()}
-      subtitle={short}
+      subtitle={posUrl}
```

Comment:
should not use the same value used in the action

Reply:
Now matching the correct url used when opening the link.